apps/system/js/utility_tray.js

```diff
@@ -4,11 +4,20 @@
 window.UtilityTray = {
   name: 'UtilityTray',
 
+  debug: 0 ? console.log.bind(console, '[UtilityTray]') : () => {},
```

Comment:
It would be nice to make the utility tray a BaseUI module now that we are cleaning it up, and use the built in debug method, but happy to move this to a follow up.